dist/index.html

```diff
@@ -9,16 +9,16 @@
 
 <body>
     <section id="listContainer">
-        <div class="heading">
-            <h1>Today's To Do</h1>
-            <img src="./images/update.png" alt="Refresh" class="icons UpdateBtn">
+        <div class="header">
+            <h1> To Do  List</h1>
+            <img src="./images/update.png" alt="Refresh" class="icons updateListBtn">
         </div>
         <div class="addList">
-            <input type="text" placeholder="Add to your list..." class="addText">
-            <img src="./images/downIcon.png" alt="Add" class="icons addBtn">
+            <input type="text" placeholder="Add to your list..." class="inputText">
```

Comment:
- [ ] [Optional] Good job on implementing the functionality of the input field! However, I would recommend making the width of the input field 100% to ensure that it takes up the full width of its container. This will help improve the visual appearance and ensure a consistent layout. To achieve this, you can update the CSS styling for the input field by setting its width property to `100%`. This will allow the input field to expand and fill the available space within its container.

Once you have made this adjustment, test the input field to ensure that it now occupies the full width. Keep up the great work! 👍🎉